blockly-rtc/server/Database.js

```diff
@@ -174,6 +174,80 @@ class Database {
       });
     });
   };
+
+  /**
+   * Add client to clients if it is not already in the table.
```

Comment:
This isn't called in this PR.

Reply:
I have now included the middleware to this PR which call these functions.